### ISSUE TITLE
Default mask ignore numbers

### ISSUE THREAD

**Issue by lolcabanon** [ionited/mask#1]
Trying to use this with MaskDefault with `{ mask: 'A9A 9A9' }` (canadian zip code) and digit values are straight up ignored.

The resulting mask is : `A AA` (which is weird since it's the reverse version of my mask).

Also, RegEx don't seem to work either... `{ mask: /[a-zA-Z][0-9][a-zA-Z] [0-9][a-zA-Z][0-9]/ }` let absolutely nothing pass!

**Comment by gabrielcursino** [comment]
Hello @lolcabanon! Thanks for the feedback!

I will take a look on this!

**Comment by gabrielcursino** [comment]
Hello @lolcabanon! This problem has been fixed with [0.6.1](https://github.com/ionited/mask/releases/tag/0.6.1) release.

In your case, now you can use:

```js
{ mask: 'A9A 9A9' }
```

or compatible regex:

```js
{ mask: /[A-Za-z]{1}[0-9]{1}[A-Za-z]{1} [0-9]{1}[A-Za-z]{1}[0-9]{1}/ }
```

**Comment by lolcabanon** [comment]
Awesome work! I just realised it works now in the REPL I liked in the other issue.

:smile:
